signac/contrib/project.py

```diff
@@ -1448,6 +1448,9 @@ class Crawler(SignacProjectCrawler):
         for doc in docs:
             yield doc
 
+    @deprecated(
+        deprecated_in="1.4", removed_in="2.0", current_version=__version__,
+        details="Uses deprecated functionality")
```

Comment:
```suggestion
You have to use the "next" version -- you can't deprecate something in a version that has already been released. 😄
```suggestion
        deprecated_in="1.5", removed_in="2.0", current_version=__version__,
        details="Access modules are deprecated.")
```

Reply:
Done :stuck_out_tongue: